.gitpod.yml

```diff
@@ -0,0 +1,20 @@
+image: nfcore/gitpod:latest
+tasks:
+  - name: install current state of nf-core/tools
+    command: |
+      NFCORE_BUILD_PATH=$HOME/nf-core_build
```

Comment:
somehow this didn't install the cli command correctly for me (`importlib.metadata.PackageNotFoundError: nf-core`).

Reply:
if I then just run `pip install --upgrade -r requirements-dev.txt -e .` it works with the current state of code as we want (using `/home/gitpod/nf-core_build/nf-core`), but somehow not if I run it with the `-t` option. 🤔